string command
int64 controller
---
int64 command_result
